CHANGES.rst

```diff
@@ -74,7 +74,7 @@ Breaking changes:
 
 - Require Python 3.7 as minimum Python version.  [maurits] [niccokunzmann]
 - icalenar now takes a ics file directly as an input
```

Comment:
typo, s/icalenar/icalendar

Reply:
@ThiefMaster would you like to create a PR?